daemon/container.go

```diff
@@ -227,6 +227,21 @@ func (container *Container) GetRootResourcePath(path string) (string, error) {
 	return symlink.FollowSymlinkInScope(filepath.Join(container.root, cleanPath), container.root)
 }
 
+func (container *Container) ExportRw() (archive.Archive, error) {
```

Comment:
Yeah, this method used to be in container_unix.go, and before that container.go.  It's been moved back as-is.